### ISSUE TITLE
Nemo file renaming

### ISSUE THREAD

**Issue by dglava** [shimmerproject/Greybird#161]
This bug affects Cinnamon's file manager Nemo.
While renaming a file, there is no clear indicator which part of the text is selected or even that the file is ready for renaming (no box surrounding it). Other themes which aren't created by the Mint team suffered from this bug too, including Arc, Zuki-themes or the official Adwaita theme (but that one won't get fixed).

Here's how it looks with Greybird:
https://webmshare.com/RAGq6
Here's how it looks with Zukitre from Zuki-themes which has a custom fix applied:
https://webmshare.com/LX51L

Would it be possible to fix this in Greybird as well?

**Comment by ochosi** [comment]
Hi,
it's possible to add a custom fix although this really doesn't make me very happy. Why can't the Cinnamon devs instead fix this in their file-manager so it works with all themes by default?
If you point me to the patch I can look into integrating it.

**Comment by dglava** [comment]
I totally agree about it being a hack and ugly workaround. It was mentioned on the Nemo issue tracker [here](https://github.com/linuxmint/nemo/issues/1150#issuecomment-213936948) and [here](https://github.com/linuxmint/nemo/issues/1173). The former one was closed, while the latter one is still open, but no words about if there are plans to change this (apart from the "bug" tag).

These two ([1](https://github.com/lassekongo83/zuki-themes/commit/df74c11b9014282ac516e0cfb1a3b25d16ee935c), [2](https://github.com/lassekongo83/zuki-themes/commit/c4fe8bb1fe6b55784b806aea407d31c2614f2388)) commits fixed it in in the Zuki-themes.

**Comment by ochosi** [comment]
Oh this looks terrible indeed... Anyway, I'll consider merging that in.

**Comment by levito** [comment]
I just prepared a PR, PTAL!
